gpytorch/gpytorchopt/bayesian_optimization_example2_oracle.py

```diff
@@ -34,7 +34,7 @@ def update_model(self, refit_hyper=True, lr=0.1, optim_steps=300, verbose=False,
         oracle_kernel_list = []
         for i in range(self.n_dims):
             oracle_kernel_list.append(gpytorch.kernels.RBFKernel(log_lengthscale_bounds=(-10, 10), active_dims=[i]))
-            oracle_kernel = gpytorch.kernels.AdditiveKernel(*oracle_kernel_list)
+            oracle_kernel = gpytorch.kernels.kernel.AdditiveKernel(*oracle_kernel_list)
```

Comment:
Was AdditiveKernel missing in the `__init__.py` of the kernels module? Or are you saying this shouldn't be part of the user-facing API? Similar comment applies in a bunch of other places.

Reply:
Yes, it was missing. I'll add it in, this fix was lazy